src/webextension/locales/fr_FR/list.ftl

```diff
@@ -0,0 +1,134 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## common
+
+document =
+  .title = Enregistrements Lockbox
+
+all-items-get-started =
+  Lorsce que vous ajoutez un compte, Il est automatiquement affiché ici.
+
+all-items-no-results = Aucun résultats
+
+item-fields-title = Nom
+item-fields-title-input
```

Comment:
You'll need to add ` =` at the end of this line for it to work with the new Fluent syntax (like you did with `document` above).

Reply:
Fixed